crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -118,27 +118,26 @@ Using the library
 
 Providing platform specific calloc/free
 ---------------------------------------
-This library facilitates the same type of memory management as regular Mbed TLS deliverables.
-This includes using internal calls to calloc/free from the library code when memory is needed.
+Just like Mbed TLS, this library calls calloc/free for memory management.
 
-The following API must be used to change the default `calloc`/`free` function:
+The `calloc`/`free` functions can be changed with the following API:
 
 .. code-block:: c
     :caption: Setting custom calloc/free
 	
     int ret;
     
-    ret = mbedtls_platform_set_calloc_free(alloc_fn, free_fn);
+    ret = mbedtls_platform_set_calloc_free(calloc_fn, free_fn);
     if (ret != 0) {
             /* Failed to set the alternative calloc/free */
             return ret;
     }
 
 .. note::
-   This API must be called prior to calling :c:func:`mbedtls_platform_setup`.
+   This API can not be called after :c:func:`mbedtls_platform_setup`.
```

Comment:
You can run mbedtls_platform_setup multiple times. The statement still holds, and shouldn't be changed

Reply:
reverted